lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -216,6 +220,12 @@ def extract_student(student, features):
                 (cohort.name for cohort in student.course_groups.all() if cohort.course_id == course_key),
                 "[unassigned]"
             )
+
+        if include_team_column:
+            student_dict['team'] = next(
+                (team.name for team in student.teams.all() if team.course_id == course_key),
+                "[unavailable]"
```

Comment:
Does this need to support i18n (I forget whether we do that for reports)? Also, is this the first time that we need to support an unavailable result? I would have thought that this pattern might have been established already. If not, it would be good to make this a constant so that future columns can reuse it.